lib/Mojo/Util.pm

```diff
@@ -379,6 +406,27 @@ sub unquote {
   return $str;
 }
 
+sub urandom_token {
+  my $length   = shift || 44;
+  my @alphabet = @{shift || ["A" .. "Z", "a" .. "z", "0" .. "9"]};
+
+  unless (@alphabet >= 2 && @alphabet <= 256) {
+    croak "alphabet size must be between 2 and 256 elements";
+  }
```

Comment:
Not covered by tests as far as i can see.

Reply:
Is a custom alphabet actually valuable for us?

Reply:
For password generation, maybe. But I've removed `urandom_token` (and custom alfabets) in favor of `urandom_urlsafe` as described in https://github.com/mojolicious/mojo/pull/2200#discussion_r1780049397